tests/nnapi/nnapi_gtest.skip.x86_64-linux.cpu

```diff
@@ -1,6 +1,4 @@
 GeneratedTests.abs_
-GeneratedTests.batch_to_space
-GeneratedTests.batch_to_space_float_1
```

Comment:
Please update this for other architecture (armv7l, aarch64)

Reply:
Applied on armv7l and aarch64